telemetry-aware-scheduling/pkg/telemetryscheduler/telemetryscheduler.go

```diff
@@ -159,15 +161,15 @@ func (m MetricsExtender) WritePrioritizeResponse(w http.ResponseWriter, result *
 //It decodes the request, checks its policy and registers it.
 //It then calls the filter logic and writes a response to the scheduler.
 func (m MetricsExtender) Filter(w http.ResponseWriter, r *http.Request) {
-	log.Print("filter request received")
+	klog.V(2).InfoS("Filter request received", "component", "extender")
 	extenderArgs, err := m.DecodeExtenderRequest(r)
 	if err != nil {
-		log.Printf("cannot filter %v", err)
+		klog.V(2).InfoS("cannot filter "+err.Error(), "component", "extender")
 		return
 	}
 	filteredNodes := m.filterNodes(extenderArgs)
 	if filteredNodes == nil {
-		log.Print("No filtered nodes returned")
+		klog.V(2).InfoS("No filtered nodes returned", "http.status", http.StatusNotFound, "component", "extender")
```

Comment:
remove the status from the log here - it's not adding anything and will just confuse the reader. http status should only be for the http client

Reply:
status removed.